tests/pytest/core/test_models.py

```diff
@@ -201,27 +146,27 @@ def test_EligibilityType_zero_expiration_reenrollment_days(model_EligibilityType
 
 
 @pytest.mark.django_db
-def test_EligibilityType_missing_reenrollment_template(model_EligibilityType_supports_expiration):
-    model_EligibilityType_supports_expiration.reenrollment_error_template = None
-    model_EligibilityType_supports_expiration.save()
+def test_EnrollmentFlow_missing_reenrollment_template(model_EnrollmentFlow_supports_expiration):
+    model_EnrollmentFlow_supports_expiration.reenrollment_error_template = None
+    model_EnrollmentFlow_supports_expiration.save()
 
     with pytest.raises(ValidationError) as exception_info:
-        model_EligibilityType_supports_expiration.full_clean()
+        model_EnrollmentFlow_supports_expiration.full_clean()
 
     error_dict = exception_info.value.error_dict
     assert len(error_dict["reenrollment_error_template"]) == 1
     assert error_dict["reenrollment_error_template"][0].message == "Required when supports expiration is True."
 
 
 @pytest.mark.django_db
-def test_EligibilityType_supports_expiration(model_EligibilityType_supports_expiration):
+def test_EnrollmentFlow_supports_expiration(model_EnrollmentFlow_supports_expiration):
     # test will fail if any error is raised
-    model_EligibilityType_supports_expiration.full_clean()
+    model_EnrollmentFlow_supports_expiration.full_clean()
 
 
 @pytest.mark.django_db
-def test_EligibilityType_enrollment_index_template():
-    new_eligibility_type = EligibilityType.objects.create()
+def test_EnrollmentFlow_enrollment_index_template():
+    new_eligibility_type = EnrollmentFlow.objects.create()
```

Comment:
Would be great to rename this variable to `new_flow` for consistency

Reply:
👍  renamed it [here](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/tests/pytest/core/test_models.py#L169)